bootstrap/ic-user-projects/create-projects-and-resources-job.yaml

```diff
@@ -305,7 +305,7 @@ spec:
             resources:
               requests:
                 storage: 5Gi
-            storageClassName: ocs-storagecluster-ceph-rbd
+            storageClassName: ocs-external-storagecluster-ceph-rbd
```

Comment:
Would be better to simply delete this line to avoid depending on a specific StorageClass name. Since we can expect or require that there is a default storage class already existing in the cluster for block storage, we do not need to tie the application to a specific storage class name.

Reply:
+1